.github/workflows/postBuildStartedComment.yml

```diff
@@ -0,0 +1,46 @@
+name: Post build has been started comment
+
+on:
+  workflow_call:
+    inputs:
+      APP_PULL_REQUEST_NUMBER:
+        description: Pull Request number from App repo for correct placement of ND app. If not specified defaults to main branch.
```

Comment:
>defaults to main branch.

Don't you need a

`default: 'main'` for this to be true then? Same with the one below.

Reply:
No, it's not needed because this workflow adds comment to PR so it needs to know concrete number. If no number is passed then it means that we don't want to add comment in this repo (it works in both App and Mobile-Expensify).